lib/util/http/github.ts

```diff
@@ -493,4 +493,24 @@ export class GithubHttp extends Http<GithubHttpOptions> {
 
     return result;
   }
+
+  public getRawFile(
```

Comment:
Add a tsdoc comment and a note that only text files are allowed. The return type indicates a string body, so when you pass a binary file url, then you'll get a Buffer as body which is higly unexpected at runtime.

So maybe rename to explpict mention and validate in function that returned body is a string.
```suggestion
  public getRawTextFile(
```

Reply:
I haven't managed to force got to return a Buffer without setting the `responseType`, though I have added a check and a test for it just to be sure.